api/src/app.ts

```diff
@@ -144,10 +144,8 @@ export const build = async (
       // Path is necessary to ensure that only one cookie is set and it is valid
```

Comment:
We don't need this comment anymore, yeah?

Reply:
Not here, no. I'll move it to where it's set.